### PR TITLE
[project] create new project

### LINKED ISSUE #99: create new project
- [x] Ask users to select a path
- [x] Input project name
- [x] choose raco template

### PR DESCRIPTION
**Pull request by dannypsnl**
NOTE: this is an initialization, find raco for any system is this next thing to do

resolve #99

### CODE REVIEW

File: project-manager.rkt [hunk 1]
```diff
@@ -70,14 +76,47 @@
...
+                    (define evt-type (send event get-event-type))
+                    (match evt-type
+                      ['list-box-dclick
+                       (send tmp-frame show #f)
+                       (match-define (list n)
+                         (send template-selection get-selections))
+                       (define path (build-path user-selected-path project-name))
+                       (cmd:run
+                        (format
+                         "/Applications/Racket\\ v8.1/bin/raco new ~a ~a"
```
Comment (dannypsnl) [inline project-manager.rkt]: FIXME

**Review by smallfont00 — APPROVED** [review]
lgtm

**Review by smallfont00 — APPROVED** [review]
lgtm again

**Comment by dannypsnl** [comment]
great thanks @shhyou

**Pull request merged**